src/Gui/RemoteDebugger.ui

```diff
@@ -89,16 +89,6 @@
          </property>
         </widget>
        </item>
-       <item row="2" column="0" colspan="2">
-        <widget class="QCheckBox" name="checkRedirectOutput">
-         <property name="text">
-          <string>Redirect output</string>
```

Comment:
could you explain what this did and why it's removed?

Reply:
It was an option in the dialog under Macro > attach to remote debugger > vscode

Since I've migrated the debugger from ptvsd to debugpy this checkbox is no longer needed.